src/Sulu/Bundle/HttpCacheBundle/Resources/config/event-subscribers.xml

```diff
@@ -22,12 +22,12 @@
             <tag name="sulu_document_manager.event_subscriber"/>
         </service>
 
-        <service id="sulu_http_cache.event_subscriber.flush" class="%sulu_http_cache.event_subscriber.flush_subscriber.class%">
+        <service id="sulu_http_cache.event_subscriber.flush" class="%sulu_http_cache.event_subscriber.flush_subscriber.class%" public="true">
```

Comment:
Since there is still PR #3428 about the cache refactoring open (*cough* @chirimoya *cough*), I've decided to go for just making these services public for the tests now.